libs/core/src/main/java/org/elasticsearch/common/util/FastMath.java

```diff
@@ -34,11 +34,12 @@
 
 package org.elasticsearch.common.util;
 
+/**
+ * Additions or modifications to this class should only come from the original org.math.plot.utils.FastMath source
+ */
 public final class FastMath {
```

Comment:
This class can be made package private now.

Reply:
👍